### DISCUSSION TITLE
Tank you for your awesome service

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1542] [Kind words ❤️]
Hello!
I want to thank you for such a useful and wonderful service! It has saved me so much nerves, protecting me from spam and various unpleasant personalities, which I didn't want to see near me at all. Your service allows me to publish my address (alias) on forums so that different people could write to me without fearing for my privacy and security, because I could delete that alias at any second! N

The only thing I do NOT like is that the user sending emails to the disabled alias does not see any "bounce". It would be nice if he got a "bounce" that the address doesn't exist. The point is that spammers and advertisers use email address validation services and if a disabled (NOT REMOVED) address tells them that the alias does not exist, it is very likely that the alias will be removed from the mailing lists with a note of invalidity.

**Comment by nguyenkims** [comment]
You can change this default behavior on https://app.simplelogin.io/dashboard/setting

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/3167780/213133309-3fa1289f-a0d2-4422-b72d-81f93b9be434.png">
